pkg/client/powervs_client.go

```diff
@@ -164,19 +158,8 @@ func NewValidatedClient(ctrlRuntimeClient client.Client, secretName, namespace,
 	if err != nil {
 		return nil, err
 	}
-
-	//Setting the region here so that corresponding power endpoint can be fetched from it
-	err = setEnvironmentVariables(regionEnvironmentalVariable, r)
-	if err != nil {
-		return nil, err
-	}
-
-	zone := resource.RegionID
-
-	err = setEnvironmentVariables(zoneEnvironmentalVariable, zone)
-	if err != nil {
-		return nil, err
-	}
+	c.region = r
+	c.zone = resource.RegionID
```

Comment:
is this really useful adding these values in the c struct? are those values anywhere from powerVSClient?